libs/application/templates/general-fishing-license/src/utils/fields.ts

```diff
@@ -0,0 +1,10 @@
+/** Stores ids of dynamic ui fields for general fishing license */
```

Comment:
This file should probably be name constants and should be moved to src/lib/constants.ts under the template with all the constants in licesnses.ts as well. The utils should be utility functions.

Reply:
There's no src/lib/constants.ts file in the GFL project. There is a src/shared/constants.ts but that only contains names of api operations. So can you explain a little further where you'd want this?

Reply:
Yep sure thing, my comment suggests making a file called constants.ts under libs/application/templates/general-fishing-license/src/libs/constants.ts where all your reusable constants should live.